controllers/util/storageconsumer.go

```diff
@@ -155,3 +156,22 @@ func (wrapper storageConsumerResourceConfigMapWrapper) SetCephFsClientProfileNam
 func (wrapper storageConsumerResourceConfigMapWrapper) SetNfsClientProfileName(name string) {
 	wrapper.Data[nfsClientProfileKey] = name
 }
+
+func GenerateStorageConsumerResourceDefaults(storageConsumerName, clientProfileName string) map[string]string {
```

Comment:
Let's try a different name:
```suggestion
func GetStorageConsumerDefaultResourceNames(storageConsumerName, clientProfileName string) map[string]string {
```